### ISSUE TITLE
From/To header incorrectly encoded

### ISSUE THREAD

**Issue by WorksDev** [swiftmailer/swiftmailer#1344]
| Q                   | A
| ------------------- | -----
| Bug report?         | **yes**
| Feature request?    | **no**
| RFC?                | **no**
| How used?           | **Standalone**
| Swiftmailer version | **6.2.7**
| PHP version         | **7.4**

### Observed behaviour
If the From/To header contains a bracket without special characters (öäß...), e.g. xxx (something) xxx, the header is not encoded correctly (unfortunately, this case is not covered by the Swift_Bug650Test). 

According to my research, the reason for this is in the Swift_Mime_Headers_AbstractHeader:

Line: 209
`if (!preg_match('/^'.self::PHRASE_PATTERN.'$/D', $phraseStr)) {`

Line: 212
`if (preg_match('/^[\x00-\x08\x0B\x0C\x0E-\x7F]*$/D', $phraseStr)) {`

Both patterns seem to contain brackets, which prevents the programme from reaching the desired structure from line 216 onwards. I would like to write a fix for this myself, but I don't understand what PHRASE_PATTERN does or where the problem lies.

Basically, OWA & Outlook in general are problematic, as they seem to cut away data from the headers mentioned or do not output them.

### Expected behaviour
Escape From/To Correctly or wrap it in "xxx (something) xxx"

### Example
Set From or To Name to "xxx (something) other"

This Problem was also reported in https://github.com/swiftmailer/swiftmailer/issues/650#issuecomment-410854299
